src/main/drivers/pwm_output.h

```diff
@@ -179,7 +180,7 @@ void pwmCompleteDshotMotorUpdate(uint8_t motorCount);
 #ifdef BEEPER
 void pwmWriteBeeper(bool onoffBeep);
 void pwmToggleBeeper(void);
-void beeperPwmInit(IO_t io, uint16_t frequency);
+void beeperPwmInit(const beeperDevConfig_t *beeperConfig);
```

Comment:
Hi martin, with the current code an optional beeper can't be used because only the tag "BEEPER" is searched in pwm output. The tag should come from the beeperDevConfig and not searched from a fixed tag. I will change this to avoid the need of the complete config, which is defined in sound_beeper.h, and only put the tag as parameter.